tasks/indexing/task_ipni.go

```diff
@@ -306,6 +313,7 @@ func (I *IPNITask) TypeDetails() harmonytask.TaskTypeDetails {
 		IAmBored: passcall.Every(5*time.Minute, func(taskFunc harmonytask.AddTaskFunc) error {
 			return I.schedule(context.Background(), taskFunc)
 		}),
+		Max: taskhelp.Max(6),
```

Comment:
Should be a config, probably with a shared max (like sdr/unseal) with the indexing task (both hammer disk reads quite a lot)

Reply:
That makes sense. I will create a common one.